recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: e2832077e258bfe906c9c0dc1664d2ba4fc0acf98bfe073c6383cb1d051b6ef0
 
 build:
-  number: 0
+  number: 1
```

Comment:
This change is for the overall build and will update the `rdkit` subpackage as well; I don't think the pin is necessary and it seems a bit out of place since it's at a higher level than the subpackage definition?